roles/ceph-common/tasks/main.yml

```diff
@@ -22,7 +22,8 @@
     ceph_stable_rh_storage_cdn_install
 
 - include: ./installs/install_on_redhat.yml
-  when: ansible_os_family == 'RedHat'
+  when: ansible_os_family == 'RedHat' and 
+        not mon_containerized_deployment
```

Comment:
This should never get triggered because of this https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/meta/main.yml#L14 , so why?